workflow/controller/operator.go

```diff
@@ -543,16 +543,17 @@ func (woc *wfOperationCtx) persistWorkflowSizeLimitErr(wfClient v1alpha1.Workflo
 	woc.markWorkflowError(err, true)
 	_, err = wfClient.Update(woc.wf)
 	if err != nil {
-		woc.log.Warnf("Error updating workflow: %v", err)
+		woc.log.Warnf("Error updating workflow with size error: %v", err)
```

Comment:
this message is indistinguishable from another error, so tweak the message

Reply:
This is great, makes finding errors much easier.